api/python/cellxgene_census/tests/README.md

```diff
@@ -1,10 +1,10 @@
 # Test README
 
-This directory contains tests of the cell-census package API, _and_ the use of the API on the
-live "corpus", i.e., data in the public cell census S3 bucket. The tests use Pytest, and have
+This directory contains tests of the cellxgene-census package API, _and_ the use of the API on the
+live "corpus", i.e., data in the public census S3 bucket. The tests use Pytest, and have
 Pytest marks to control which tests are run.
 
-Tests can be run in the usual manner. First, ensure you have cell-census installed, e.g., from the top-level repo directory:
+Tests can be run in the usual manner. First, ensure you have cellxgene-census installed, e.g., from the top-level repo directory:
 
 > pip install -e ./api/python/cell_census/
 
```

Comment:
l don't know why but I can't suggest in line 13 which needs to be changed as well

From
```
> pytest ./api/python/cell_census/
```

To
```
> pytest ./api/python/cellxgene_census/
```